scripts/tests/test_zkai_attention_kv_stwo_binary_typed_proof_accounting_gate.py

```
@@ -0,0 +1,170 @@
+
+    def test_rejects_bool_encoded_metric(self):
+        summary = cli_summary()
+        payload = gate.build_payload(summary)
+        payload["profile_rows"][0]["local_binary_accounting"]["records"][0]["item_count"] = True
+        payload["payload_commitment"] = gate.payload_commitment(payload)
+        self.assert_rejects(payload, summary, "item_count must be an integer")
+
+    def test_rejects_cli_schema_drift(self):
+        summary = cli_summary()
+        summary["schema"] = "wrong"
+        with self.assertRaisesRegex(gate.BinaryTypedProofAccountingGateError, "CLI schema drift"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)